tests/hamcrest_unit_test/core/is_test.py

```diff
@@ -65,7 +65,6 @@ def test_description_uses_equal_to(arg, identity, desc):
 @pytest.mark.parametrize('arg, identity', (
     ('A', str),
     (1, int),
-    only_py2((OldClass(), OldClass)),
```

Comment:
I think we'll be cutting a 1.10 branch and releasing from that, and Python 2 won't be supported. Is this fix related to #123?

Reply:
Yes it is fixing #123, at least on Debian/Ubuntu.
Thanks for the new branch. In Debian/Ubuntu we need Python 2 support temporarily because we are in the middle removing Python 2 support from reverse dependencies.

Reply:
Ah, in that case, perhaps we need a Python 2 release. I'll talk to @offbyone.

Reply:
@brunns Thanks, but for Debian/Ubuntu it is enough to have the patches accepted. We also hope to finish the Python 2 removal and carry the patch in packaging only, so from our POV a new Python 2 release is not needed.

Reply:
I'm not averse to releasing patches on the 1.x line for legacy Python support, as long as they're from PRs. I'm not going to do much work that way myself, but...

Anyway

Reply:
I don't expect you to! I'm happy to do it myself, but I will need credentials, and ideally a very brief overview of the process.